source/fundamentals/gridfs.txt

```diff
@@ -1,86 +1,84 @@
-.. uses gridfs.rst
+.. _pymongo-gridfs:
 
-GridFS Example
-==============
+GridFS
+======
 
-.. code-block:: python
-
-  from pymongo import MongoClient
-
-  client = MongoClient()
-  client.drop_database("gridfs_example")
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
 
-This example shows how to use ``gridfs`` to store large binary
-objects (e.g. files) in MongoDB.
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: binary large object, blob, file, storage
 
-.. seealso:: The API docs for ``gridfs``.
-
-.. seealso:: `This blog post
-   <http://dirolf.com/2010/03/29/new-gridfs-implementation-for-pymongo.html>`_
-   for some motivation behind this API.
+This guide shows how to use GridFS to store and retrieve large
+in MongoDB.
 
 Setup
 -----
 
-We start by creating a ``~gridfs.GridFS`` instance to use:
+Each GridFS instance operates on a specific MongoDB database.
```

Comment:
S: monospace since this seems to be talking about the GridFS instance rather that just GridFS in general

```suggestion
Each ``GridFS`` instance operates on a specific MongoDB database.
```